cla_common/constants.py

```diff
@@ -515,9 +515,6 @@
 )
 
 OPERATOR_HOURS = {
-    "weekday": (datetime.time(9, 0), datetime.time(17, 0)),
```

Comment:
Have these changes been tested against cla_backend? I Think removal of these date/times might break existing tests in cla_backend

Reply:
The backend tests will need updating, yes. 
Good point on the Christmas hours. I'll leave one pair in, for the tests to have something to test (although we probably should be mocking anyway).